### ISSUE TITLE
UnicodeDecodeError: 'utf-8' codec can't decode byte XXX in position YYY: ZZZ

### ISSUE THREAD

**Issue by williballenthin** [mandiant/capa#707]
```
   1   │ Traceback (most recent call last):
   2   │   File "/usr/lib/python3.8/runpy.py", line 194, in _run_module_as_main
   3   │     return _run_code(code, main_globals, None,
   4   │   File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
   5   │     exec(code, run_globals)
   6   │   File "/home/user/code/capa-pub/capa/main.py", line 991, in <module>
   7   │     sys.exit(main())
   8   │   File "/home/user/code/capa-pub/capa/main.py", line 899, in main
   9   │     extractor = get_extractor(
  10   │   File "/home/user/code/capa-pub/capa/main.py", line 464, in get_extractor
  11   │     vw = get_workspace(path, format, sigpaths)
  12   │   File "/home/user/code/capa-pub/capa/main.py", line 412, in get_workspace
  13   │     vw = viv_utils.getWorkspace(path, analyze=False, should_save=False)
  14   │   File "/home/user/env/lib/python3.8/site-packages/viv_utils-0.6.5-py3.8.egg/viv_utils/__init__.py", line 101, in getWorkspace
  15   │     vw.loadFromFile(fp)
  16   │   File "/home/user/env/lib/python3.8/site-packages/vivisect/__init__.py", line 2676, in loadFromFile
  17   │     fname = mod.parseFile(self, filename, baseaddr=baseaddr)
  18   │   File "/home/user/env/lib/python3.8/site-packages/vivisect/parsers/pe.py", line 36, in parseFile
  19   │     return loadPeIntoWorkspace(vw, pe, filename, baseaddr=baseaddr)
  20   │   File "/home/user/env/lib/python3.8/site-packages/vivisect/parsers/pe.py", line 366, in loadPeIntoWorkspace
  21   │     for rva, lname, iname in pe.getImports():
  22   │   File "/home/user/env/lib/python3.8/site-packages/PE/__init__.py", line 452, in getImports
  23   │     return self.imports
  24   │   File "/home/user/env/lib/python3.8/site-packages/PE/__init__.py", line 1172, in __getattr__
  25   │     self.parseImports()
  26   │   File "/home/user/env/lib/python3.8/site-packages/PE/__init__.py", line 796, in parseImports
  27   │     self.imports = self.parseImportTable(x, irva, is_imports=True)
  28   │   File "/home/user/env/lib/python3.8/site-packages/PE/__init__.py", line 878, in parseImportTable
  29   │     funcname = ibn.Name
  30   │   File "/home/user/env/lib/python3.8/site-packages/vstruct/__init__.py", line 420, in __getattr__
  31   │     return r.vsGetValue()
  32   │   File "/home/user/env/lib/python3.8/site-packages/vstruct/primitives.py", line 601, in vsGetValue
  33   │     s = self._vs_value.split(b'\x00')[0].decode('utf-8')
  34   │ UnicodeDecodeError: 'utf-8' codec can't decode byte 0x98 in position 0: invalid start byte
```

**Comment by williballenthin** [comment]
need either:
  - viv to handle this case robustly, or
  - a more specific bug from viv that we and handle and format

**Comment by williballenthin** [comment]
the file may be corrupt or packed if the import table contains non-ASCII

**Comment by williballenthin** [comment]
![image](https://user-images.githubusercontent.com/156560/130281006-6bf0c9ef-c08e-4373-a7f2-6d100104981e.png)
IDA is surprisingly forgiving

**Comment by williballenthin** [comment]
https://github.com/vivisect/vivisect/issues/440 and https://github.com/vivisect/vivisect/pull/441

**Comment by williballenthin** [comment]
closed in #731

**Comment by mr-tz** [comment]
updated vivisect to v1.0.5 in #786, this should be fixed
